README.md

```diff
@@ -1,3 +1,135 @@
+```js
+{
+    chrome: {
+        path: 'C:\\ProgramFiles\\...\\chrome.exe', 
+        cmd: '--new-window', 
+        macOpenCmdTemplate: 'open -n -a "{{{path}}}" --args {{{pageUrl}}} {{{cmd}}}' 
+    },
+
+    firefox: {
+        path: 'C:\\ProgramFiles\\...\\firefox.exe',  
+        cmd: '-new-window', 
+        macOpenCmdTemplate: 'open -a "{{{path}}}" {{{pageUrl}}} --args {{{cmd}}}'
+    }
+}
+```
+
+##open
+Opens the web page in a new instance of the browser.
+```js
+async open(browserInfo, pageUrl)
+```
+**browserInfo** – Object (required). Provides information on the browser where the web page should be opened. See the object structure above.
+
+**pageUrl** – String (required). Specifies the web page URL.
+
+##close
+Closes the browser window where the specified web page is opened.
+```
+async close(pageUrl) 
+```
+**pageUrl** – String (required). Specifies the URL of the web page opened in the browser.
+
+
+##screenshot
+Takes a screenshot of the browser window where the specified web page is opened.
+```
+async screenshot(pageUrl, screenshotPath) 
+```
+**pageUrl** -  String (required). Specifies the URL of the web page opened in the browser.
+
+**screenshotPath** – String (required). Specifies the full path to the screenshot file. For example, *D:\Temp\chrome-screenshot.jpg*.
+
+
+##resize
+The function has two versions. The first one changes the browser window size to the new width and height.
+```
+async resize(pageUrl, width, height) 
+```
+**pageUrl** – String (required). Specifies the URL of the web page opened in the browser.
+
+**width** – Integer (required). Specifies the new window width in pixels.
+
+**height** – Integer (required). Specifies the new window height in pixels.
+
+
+The other version of the **resize** function allows you to change the browser window size according to the screen size of the target device.
+```
+async resize(pageUrl, deviceName, orientation) 
+```
+**pageUrl** – String (required). Specifies the URL of the web page opened in the browser.
+
+**deviceName**  – String (required). Specifies the name of the target device. You can use the values specified in the **Device Name** column of [this table](http://viewportsizes.com/).
+
+**orientation** - String (optional). Specifies the device orientation: *"portrait"* or *"landscape"* (by default).
+
+#Testing
+
+To run automated tests:
+```
+$ npm test
+```
+Since the module functionality depends on browsers available on a testing machine and you cannot predict expected returned values for some functions, the automated tests cover only a part of the functionality.  
+To test all the functions provided by the module, use the playground. To run it, execute the gulp task corresponding to your operating system:
+```
+$ gulp run-playground-win 
+$ gulp run-playground-mac
+$ gulp run-playground-linux
+```
+This will open the Playground web page at [http://localhost:1334/](http://localhost:1334/) , where you can manually check if the functions work correctly.
```

Comment:
Do we really need to give a link to localhost here? Especially given that this sentence states that the page is opened automatically. Maybe use plain text?
Links to localhost look weird to me,